moshi_mlx/pyproject.toml

```diff
@@ -19,6 +19,9 @@ license = {text = "MIT"}
 dynamic = ["version"]
 readme = "README.md"
 
+[project.scripts]
+moshi-local = "moshi_mlx.local:_main"
+moshi-local-web = "moshi_mlx.local_web:main"
```

Comment:
Not sure if these are the desired names, but I kept to the module naming convention.

Reply:
yes sounds good